src/main/resources/update/gitbucket-core_4.12.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changeSet>
+  <createTable tableName="RELEASE">
+    <column name="USER_NAME" type="varchar(100)" nullable="false"/>
+    <column name="REPOSITORY_NAME" type="varchar(100)" nullable="false"/>
+    <column name="RELEASE_ID" type="int" nullable="false"/>
+    <column name="NAME" type="varchar(100)" nullable="false"/>
+    <column name="TAG" type="varchar(100)" nullable="false"/>
+    <column name="AUTHOR" type="varchar(100)" nullable="false"/>
+    <column name="CONTENT" type="text" nullable="true"/>
+    <column name="IS_DRAFT" type="boolean" nullable="false"/>
+    <column name="IS_PRERELEASE" type="boolean" nullable="false"/>
+    <column name="REGISTERED_DATE" type="datetime" nullable="false"/>
+    <column name="UPDATED_DATE" type="datetime" nullable="false"/>
+  </createTable>
+
+  <addPrimaryKey constraintName="IDX_RELEASE_PK" tableName="RELEASE" columnNames="USER_NAME, REPOSITORY_NAME, RELEASE_ID"/>
+  <addUniqueConstraint constraintName="IDX_RELEASE_UNIQ" tableName="RELEASE" columnNames="USER_NAME, REPOSITORY_NAME, TAG"/>
+  <addForeignKeyConstraint constraintName="IDX_RELEASE_FK0" baseTableName="RELEASE" baseColumnNames="USER_NAME, REPOSITORY_NAME" referencedTableName="REPOSITORY" referencedColumnNames="USER_NAME, REPOSITORY_NAME"/>
+
+  <createTable tableName="RELEASE_ID">
```

Comment:
I found that release has id field in [GitHub release API](https://developer.github.com/v3/repos/releases/#get-a-single-release).
It maybe useful for implementing compatible-api.
and I misunderstood id numbering, because example shows id=1, each repository starts releaseId=1, 2, ... but it is not true. I checked now.

I will rewrite AUTO_INCREMENT based release ID in this weekend.